.github/workflows/delete-v2-review-app.yml

```diff
@@ -0,0 +1,110 @@
+name: Delete v2 Review App
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+
+jobs:
+  delete-v2-review-app:
+    name: Delete v2 Review App ${{ github.event.pull_request.number }}
+    concurrency: deploy_v2_review_${{ github.event.pull_request.number }}
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'deploy_v2') || contains(github.event.pull_request.labels.*.name, 'dependencies') }}
+    runs-on: ubuntu-latest
+    environment: review_aks
+    steps:
+      - name: Exit whole workflow if wait was not successful
+        if: ${{ steps.wait_for_deployment.outputs.conclusion != '' && steps.wait_for_deployment.outputs.conclusion != 'success' }}
```

Comment:
This step looks redundant as the `wait_for_deployment` step referenced in the condition doesn't exist